docs/reference/method/MongoDBCollection-mapReduce.txt

```diff
@@ -2,6 +2,8 @@
 MongoDB\\Collection::mapReduce()
 =================================
 
+.. versionadded:: 1.2
+
```

Comment:
I added versionadded for MongoDBCollection-mapReduce.txt here.

Reply:
My mistake. I scrolled right past this.